Sources/PointFree/Blog/BlogMiddleware.swift

```diff
@@ -14,7 +14,7 @@ func blogMiddleware(
   let (user, subscriberState, route, subRoute) = lower(conn.data)
 
   switch subRoute {
-  case .feed(.atom):
```

Comment:
it wasn't really necessary for the blog to have this sub-enum in the route so I just removed it